orc8r/cloud/go/services/ctraced/obsidian/handlers/trace_handlers.go

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Unless you expect to need more handlers, I would just move the to the `handlers.go` file and unexport the functions.